mix.exs

```diff
@@ -44,7 +44,7 @@ defmodule Hexpm.MixProject do
       {:ex_aws, "~> 2.0"},
       {:ex_machina, "~> 2.0"},
       {:eqrcode, "~> 0.1.6"},
-      {:goth, "~> 1.2"},
+      {:goth, "~> 1.3-dev", github: "peburrows/goth"},
```

Comment:
Is this just a test PR while we wait for 1.3 to ship or are you planning to use the pre-release version?

Reply:
I plan to ship 1.3.0-rc.0 this Tuesday and wait at least couple weeks for folks to try it out before shipping stable.

I still have to test this on hexpm staging but yeah I’d like to ship RC to prod too.

Reply:
If you have any feedback about the new API pls let me know btw :D

Reply:
looks good on staging. just to be on safer side and since we use goth in all the other services, I'm actually going to roll it to prod on HexDocs and others first and to hexpm at the end when everything looks good.